### ISSUE TITLE
MultiWii Serial Protocol

### ISSUE THREAD

**Issue by UncleRus** [UncleRus/min_raw_osd#3]
Add MSP support

**Comment by tracernz** [comment]
Hi @UncleRus,

This would be a nice addition, and it looks like you've already made good progress. MSP is the preferred OSD protocol for us at [dRonin](/d-ronin/dRonin) because it is stable between releases, whereas UAVTalk is not. Let us know if you need any assistance, either on here, or on IRC (#dronin on Freenode).

**Comment by UncleRus** [comment]
Hi @tracernz !

MSP support will be added in week or two. But I personally prefer UAVTalk because of:
- its implementation in latest MultiOSD works pretty stable with dRonin 
- user can easily control telemetry update rate by UAVObject settings in GCS
- adding support for new UAVTalk versions is simple and fast
- MSP provides less telemetry data than UAVTalk (e.g. no flight modes, no mag data source, etc.)

By the way, I'm thinking about GCS plugin for MultiOSD: firmware, settings, screens, fonts and so on.

**Comment by tracernz** [comment]
Yes, MSP is a bit of a mess in parts, especially flight modes. GCS plugin is a nice idea. Would be good to discuss that more with you in future.
